server/server.js

```diff
@@ -4,6 +4,7 @@ const morgan = require('morgan');
 const bodyParser = require('body-parser');
 const app = express();
 const authRouter = require('./auth/auth-router.js');
+const { apiRouter } = require('./api/api-router.js');

```

Comment:
if it's the only thing you're exporting, make it default and require it without {}

Reply:
done